bin/tronctl

```diff
@@ -300,8 +313,8 @@ def control_objects(args: argparse.Namespace):
             # and changing the API would be painful, so instead we call skip + publish separately from the client
             # (i.e., this file) to implement this functionality
             if request(
-                url=urljoin(args.server, tron_id.url),
-                data={"command": "skip"},
+                    url=urljoin(args.server, tron_id.url),
+                    data={"command": "skip"},
```

Comment:
Any reason we need to add another indent here?

Reply:
we can probably get black to format this file - i bet it's skipping it since it has no extension